meinberlin/apps/budgeting/phases.py

```diff
@@ -77,6 +77,21 @@ class RatingPhase(phases.PhaseContent):
     features = {"rate": (models.Proposal,)}
 
 
+class BudgetingTwoSupportPhase(phases.PhaseContent):
+    app = apps.Config.label
+    phase = "support_budgeting_two"
+    view = views.ProposalListView
+
+    name = _("Which of the submitted proposals would you like to support?")
+    description = _(
+        "Support the submitted proposals for the participatory budgeting, "
+        "that you like."
```

Comment:
I don't think that's correct english, the comma seems unnecessary. Any thoughts?

Reply:
Yeah, I would also say comma not needed, lets just remove it?